.gitignore

```
@@ -0,0 +1,9 @@
+*.o
+.DS_Store
+.*.swp
+*.orig
+*.rej
+.vscode*
+.env
+.env.*
+*.patch
```

```suggestion
/.env
/.env.*
/*.patch
```

I think we should only ignore these suffix in root directory, not all.
Otherwise like `net/foo/patches/bar.patch` will be ignored too.

---

yes correct. That is why i specified ONLY in root directory. Not sure about env files tho

---

Oops! 🤦 Sorry for that. It is quite obvious that I havent tested it properly before submitting it to the upstream.